xml/architecture.xml

```diff
@@ -15,18 +15,18 @@
   </fingerprint>
   <fingerprint pattern="PowerPC|PPC|POWER|ppc">
     <description>PowerPC</description>
-    <param pos="0" name="os.arch" value="ppc"/>
+    <param pos="0" name="os.arch" value="PowerPC"/>
   </fingerprint>
   <fingerprint pattern="SPARC" flags="REG_ICASE">
     <description>SPARC</description>
-    <param pos="0" name="os.arch" value="sparc"/>
+    <param pos="0" name="os.arch" value="Sparc"/>
   </fingerprint>
   <fingerprint pattern="mips" flags="REG_ICASE">
     <description>MIPS</description>
-    <param pos="0" name="os.arch" value="mips"/>
+    <param pos="0" name="os.arch" value="MIPS"/>
   </fingerprint>
   <fingerprint pattern="arm" flags="REG_ICASE">
     <description>ARM</description>
```

Comment:
We recently added 'aarch64' as well for 64-bit ARM.

Reply:
Yes, I think it would be good to add 'aarch64' for ARM64 (Ref: https://github.com/rapid7/assessment-core/blob/0007d245ded9275ef495800227b155d822bc0237/fingerprinting-core/src/main/java/com/rapid7/fingerprint/system/IArchitectureConstants.java#L18)
Regarding the case, from my understanding, I don't think there's any dependency from here. But, I think it would be better to keep it alike so we won't have to check multiple places for any dependency and will be consistent.

Reply:
You might want to delete this private repository URL from the comment. I went ahead and pushed arm64/aarch64 matches to "ARM64".

Reply:
Looks like Nexpose has arm64/aarch64 mapping to "aarch64" not to "ARM64" with that recent update

Reply:
Yes, In nexpose, ARM64 is mapped to "aarch64".

Reply:
Does this impact this PR?

Reply:
Ex: all patterns used in this xml match to an uppercase value. Do you mean Nexpose maps only aarch64 to lowercase?

Reply:
Yes, we had recently added this one in lowercase.

Reply:
That sounds like a Nexpose bug =D 

Does this PR need any changes before it can be merged?